podcasts/Onboarding/LoginCoordinator.swift

```diff
@@ -177,6 +186,7 @@ extension LoginCoordinator: SyncSigninDelegate, CreateAccountDelegate {
             return
         }
 
+        Analytics.track(.userSignInFailed, properties: ["source": socialAuthProvider?.rawValue ?? "password", "error_code": (error as NSError).code])
```

Comment:
We also have the `user_account_creation_failed` event. However, there's no way to know if the user is signing in or creating an account for SSO. So I decided to use `userSignInFailed`.